### DISCUSSION TITLE
Leveling layouts

### DISCUSSION THREAD

**Discussion by Bapelsinen** [Lailloken/Exile-UI#272] [Ideas & Feedback]
Hey!
Just wondering if it's possible to change/add to the layouts feature since some layouts seem to be missing when compared to poehub.gg maybe they are outdated but they have very descriptive layout images so would like to change.

Thanks for your time and work on this tool!

**Comment by Lailloken** [comment]
Yes, you can replace them if you want, but the file-names use the game-internal area-IDs, so it may be tricky to tell which is which. They are stored in `img\GUI\leveling tracker\zones` and the file names are structured like this: `part` _ `act` _ `zone` _ `subzone` `possible layout`.
For example, the area `The Coast` (when leaving Lioneye's Watch) is `1_1_2` and has three possible layouts, so `1_1_2 1`, `1_1_2 2`, `1_1_2 3`.

Keep in mind that they will be overwritten with every update, so you should probably keep your custom ones in a folder somewhere outside the script in order to copy them back in after every update.

**Reply by Bapelsinen** [reply]
Okay Thanks!
